employee.rb

```diff
@@ -0,0 +1,18 @@
+class Employee
+  attr_reader :name
+  attr_reader :salary
```

Comment:
Nice use of `reader` instead of `accessor` when the outside world doesn't need to make changes.